secretsmanager.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_secretsmanager_secret" "db_master_user" {
+  name = "ranger-db-master-user"
+}
+
+resource "aws_secretsmanager_secret_version" "db_master_user" {
+  secret_id     = "${aws_secretsmanager_secret.db_master_user.id}"
+  secret_string = "${jsonencode(map("username",var.db_master_username,"password",random_string.db_master_password.result))}"
+}
+
+resource "random_string" "db_audit_password" {
+  length  = 16
+  special = false
+}
+
+resource "aws_secretsmanager_secret" "db_audit_user" {
+  name = "ranger-db-audit-user"
+}
+
+resource "aws_secretsmanager_secret_version" "db_audit_user_user" {
+  secret_id     = "${aws_secretsmanager_secret.db_audit_user.id}"
+  secret_string = "${jsonencode(map("username",var.db_audit_username,"password",random_string.db_audit_password.result))}"
```

Comment:
We should add in CHANGELOG or README that the tfstate file of this module will have `secrets` and recommend to encrypt and set proper permissions on tfstate file.

Reply:
State file generally contain secrets and need to be protected, this is not specific to using SecretsManager, you can see from following output, db password is saved in state file.

snippet form "terraform state pull"
                    "primary": {
                        "id": "ranger-cluster",
                        "attributes": {
                            "apply_immediately": "true",
                            "master_password": "xxxxxxxxxxxxxxxxx",
                            "master_username": "ranger",
                            "port": "3306",